explorer/Explorer.tsx

```diff
@@ -351,20 +361,72 @@ export class Explorer
         } = grapherConfigFromExplorer
         const grapherConfigFromExplorerOnlyGrapherProps = omit(
             grapherConfigFromExplorer,
-            ["mapTargetTime"]
+            [
+                "yIndicatorIds",
+                "xIndicatorId",
+                "colorIndicatorId",
+                "sizeIndicatorId",
+                "mapTargetTime",
+            ]
         )
 
-        const hasGrapherId = grapherId && isNotErrorValue(grapherId)
+        let creationMode: ExplorerChartCreationMode
+        if (grapherId && isNotErrorValue(grapherId))
+            creationMode = ExplorerChartCreationMode.FromGrapherId
+        else if (yIndicatorIds)
+            creationMode = ExplorerChartCreationMode.FromIndicatorIds
+        else
+            creationMode =
+                ExplorerChartCreationMode.FromExplorerTableColumnSlugs
 
-        const grapherConfig = hasGrapherId
-            ? this.grapherConfigs.get(grapherId!) ?? {}
-            : {}
+        const grapherConfig =
+            creationMode === ExplorerChartCreationMode.FromGrapherId
+                ? this.grapherConfigs.get(grapherId!) ?? {}
+                : {}
 
         const config: GrapherProgrammaticInterface = {
             ...grapherConfig,
             ...grapherConfigFromExplorerOnlyGrapherProps,
+            bakedGrapherURL: BAKED_GRAPHER_URL,
             hideEntityControls: this.showExplorerControls,
-            manuallyProvideData: tableSlug ? true : false,
+            manuallyProvideData:
+                creationMode ===
+                ExplorerChartCreationMode.FromExplorerTableColumnSlugs,
+        }
+
+        if (creationMode === ExplorerChartCreationMode.FromIndicatorIds) {
+            const dimensions = config.dimensions ?? []
+            if (yIndicatorIds) {
+                const yIndicatorIdsList = yIndicatorIds
+                    .split(" ")
+                    .map((item) => parseInt(item))
```

Comment:
Probably best to specify a radix of 10, just to be safe:
```suggestion
                    .map((item) => parseInt(item, 10))
```